activation/activation.go

```diff
@@ -52,12 +52,21 @@ func NewBuilder(nodeId block.NodeId, db database.DB, meshdb *mesh.MeshDB, net Br
 }
 
 func (b *Builder) PublishActivationTx(nipst *nipst.NIPST) error {
-	seq := b.GetLastSequence(b.nodeId)
+	var seq uint64
 	prevAtx, err := b.GetPrevAtxId(b.nodeId)
-	if seq > 0 && err != nil {
-		log.Error("cannot find prev ATX for nodeid %v ", b.nodeId)
-		return err
+	if prevAtx == nil {
+		log.Info("previous ATX not found")
+		prevAtx = &block.EmptyAtx
+		seq = 0
+	} else  {
+		seq = b.GetLastSequence(b.nodeId)
```

Comment:
doesn't it make more sense to pass the `atxID` (and rename this `GetSequence`) instead of looking for the `prevAtx` again inside this method? Not a big deal, if you disagree.